ftw/noticeboard/tests/test_add_content.py

```diff
@@ -19,25 +24,39 @@ def test_add_noticeboard(self, browser):
 
     @browsing
     def test_add_noticecategory(self, browser):
-        self.grant('Manager')
-
         noticeboard = create(Builder('noticeboard').titled(u'Noticeboard'))
 
         browser.login().visit(noticeboard)
         factoriesmenu.add('NoticeCategory')
-        browser.fill({'Title': u'This is a Category'})
+        browser.fill({'Title': u'This is a Category', 'Terms and Conditions': 'Anything'})
         browser.find_button_by_label('Save').click()
         self.assertEquals(u'This is a Category', plone.first_heading())
 
     @browsing
     def test_add_notice(self, browser):
-        self.grant('Manager')
+        noticeboard = create(Builder('noticeboard').titled(u'Noticeboard'))
+        category = create(Builder('noticecategory')
+                          .having(conditions=RichTextValue('Something'))
+                          .titled(u'Category')
+                          .within(noticeboard))
 
+        browser.login().visit(category)
+        factoriesmenu.add('Notice')
+        browser.fill({'Title': u'This is a Notice', 'Terms and Conditions': True})
```

Comment:
Generally yes. But in this case this is tested by ftw.testbrowser and everything else ist default plone so far.

Once there are some custom views and listings for Notices I'll add more realistic test data,  because then I can screw up 😆